### PR TITLE
Updating and industrialising plugin 

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
*Au programme*

- CI with travis and coveralls
- Update to seneca 8+
- tests with lab
- gitignore

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
@mirceaalexandru I can add travis and CI badge if you are up to.

(but you will need to active the repo on travis and coveralls)

File: package.json [hunk 1]
```diff
@@ -16,12 +18,16 @@
     "plugin"
   ],
   "author": "Mircea Alexandru",
```
Comment (mirceaalexandru) [inline package.json]: @AdrieanKhisbe please add yourself as contributor.
I will merge it tomorrow.

Thank you

**Comment by AdrieanKhisbe** [comment]
@mirceaalexandru *voilà* :)

**Comment by AdrieanKhisbe** [comment]
do you agree we drop support of 0.10, and 0.12?
To have `--harmony`, hence `const` and co?

**Comment by mirceaalexandru** [comment]
Yes, it makes sense.

Merci beaucoup :)

On Thu, Dec 10, 2015 at 5:34 PM, Adrien Becchis <notifications@github.com>
wrote:

> do you agree we drop support of 0.10, and 0.12?
> To have --harmony, hence const and co?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirceaalexandru/seneca-cron/pull/7#issuecomment-163660513>
> .
>



-- 

*malex*

**Comment by AdrieanKhisbe** [comment]
you're welcome :)

**Comment by blainsmith** [comment]
How's about a better README documenting the commands :-)

**Comment by AdrieanKhisbe** [comment]
I would have done it but @mirceaalexandru said he'll take care of it tomorrow :)

By the way @blainsmith, do you know if there is a *standard* way to document the seneca commands?
(and the seneca plugin options)

**Pull request merged**